task/10-katas-1-tasks.js

```diff
@@ -88,7 +185,58 @@ function* expandBraces(str) {
  *
  */
 function getZigZagMatrix(n) {
-    throw new Error('Not implemented');
+  let result = Array.from({ length: n }, x => Array(n));
+  let num = 0,
+    rowCount = 0,
+    colCount = 0;
+  for (let i = 0; i < result.length * 2; i++) {
+    if (colCount % 2 != 0 && rowCount == 0) {
```

Comment:
"operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
'0' != 0
and '0' !== 0)"

"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"